src/index.ts

```diff
@@ -1676,6 +1680,65 @@ class Spanner extends GrpcService {
   static pgJsonb(value): PGJsonb {
     return new codec.PGJsonb(value);
   }
+
+  /**
+   * @typedef IProtoMessageParams
+   * @property {object} value Proto Message Value. Serialized Buffer or message object.
+   * @property {string} fullName Proto message Full Qualified Path Name.
+   * @property {Function} [messageFunction] Function generated by protobufs which
+   * has helper methods to deserializing and serializing messages.
+   */
+  /**
+   * Helper function to get a Cloud Spanner proto Message object.
+   *
+   * @param {IProtoMessageParams} value The proto message value params in the format of
+   *     @code{IProtoMessageParams}
+   * @returns {ProtoMessage}
+   *
+   * @example
+   * ```
+   * const {Spanner} = require('@google-cloud/spanner');
+   * const protoMessage = Spanner.protoMessage({
+   *   value: singerInfo,
+   *   messageFunction: music.SingerInfo,
+   *   fullName: "spanner.examples.music.SingerInfo"
+   * });
+   * ```
+   */
+  static protoMessage(value: IProtoMessageParams): ProtoMessage {
+    return new codec.ProtoMessage(value);
+  }
+
+  /**
+   * @typedef IProtoEnumParams
+   * @property {string | number} value Proto Enum Value. Enum String Constant
+   * or Integer Constant.
+   * @property {string} fullName Proto Enum Full Qualified Path Name.
+   * @property {object} [enumObject] An object which have mapping from enum
+   * string to integer constant. User can use it from typings/js generated by
+   * protobufjs.
```

Comment:
Maybe simplify this a bit?

```suggestion
   * @property {object} [enumObject] An enum object generated by protobufjs.
```